packages/opentelemetry-resources/src/detectors/BrowserDetector.ts

```diff
@@ -61,5 +62,18 @@ class BrowserDetector implements Detector {
   }
 }
 
+//Add Browser related attributes to resources
+export function getBrowserAttributes():ResourceAttributes {
+  const browserAttribs:ResourceAttributes={};
+  const userAgentData=(navigator as any).userAgentData;
```

Comment:
Would you mind adding types for the `navigator` and `userAgentData`?

Reply:
navigator is a big object from the browser , and the type Navigator  doesn't currently have the userAgentData. I think it is found in the experimental version.